src/core/Tooltip/TooltipContent/TooltipContent.tsx

```diff
@@ -0,0 +1,91 @@
+import React, {
+  Component,
+  forwardRef,
+  RefObject,
+  ReactNode,
+  MouseEvent,
+  KeyboardEvent,
+} from 'react';
+import { default as styled } from 'styled-components';
+import { SuomifiThemeProp, SuomifiThemeConsumer } from '../../theme';
+import { HtmlButton } from '../../../reset/HtmlButton/HtmlButton';
+import { Icon } from '../../Icon/Icon';
+import { baseStyles } from './TooltipContent.baseStyles';
+
+const baseClassName = 'fi-tooltip';
```

Comment:
Might be good idea that we will have some className for the "component", so we would have `fi-tooltip_content`.

Reply:
TooltipContent now has the `fi-tooltip_content` added.